src/index.spec.ts

```diff
@@ -1,12 +1,163 @@
+import fetchMock from 'jest-fetch-mock';
+fetchMock.enableMocks();
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+describe('is integer', () => {
+  test("it should return false if the argument isn't a integer", () => {
+    const actual1 = '123';
+    const actual2 = '';
+    const actual3 = null;
+    const actual4 = Number.NaN;
+
+    const expected = false;
+
+    expect(isInteger(actual1)).toBe(expected);
+    expect(isInteger(actual2)).toBe(expected);
+    expect(isInteger(actual3)).toBe(expected);
+    expect(isInteger(actual4)).toBe(expected);
+  });
+  test('it should return true if the argument is a integer', () => {
+    const actual = 123;
+    const expected = true;
+
+    expect(isInteger(actual)).toBe(expected);
+  });
+});
+
+describe('to lower case', () => {
+  test("it should return a message 'Please provide a string'", () => {
+    const expected = 'Please provide a string';
+
+    expect(toLowerCase('')).toBe(expected);
+    expect(toLowerCase(null)).toBe(expected);
+  });
+  test('it should convert sent string to lower case', () => {
+    expect(toLowerCase('I <3 JEst')).toBe('i <3 jest');
+  });
+});
+
+describe('remove duplicates from array', () => {
+  test("it should return Error if argument isn't a array", () => {
+    const actual: any = 123;
+
+    const expected = Error('please provide an array of numbers or strings');
+
+    const result = () => removeDuplicatesFromArray(actual);
+
+    expect(result).toThrow(expected);
```

Comment:
What kind of error are we validating here? If we now the specific errors to be threw, we need to validate that specific error